core/sync_assets.py

```diff
@@ -0,0 +1,57 @@
+import psycopg2
+import sys
+from config_loader import load_config
+from Historic_Crypto import Cryptocurrencies
+
+def connect_db():
+    env_settings = load_config()
+    # Connect to the database
+    try:
+        conn = psycopg2.connect(
+            dbname=env_settings['DBNAME'],
+            user=env_settings['DBUSER'],
+            password=env_settings['DBPASSWORD'],
+            host=env_settings['DBHOST'],
+            port=env_settings['DBPORT']
+        )
+        print("Database connection successful.")
+        return conn
+    except Exception as e:
+        print(f"Error connecting to the database: {e}")
+        sys.exit(1)
+
+def update_or_create_asset(conn, name, symbol='$', asset_type='x'):
+    # Create a cursor
+    with conn.cursor() as cursor:
+        # First, try to update the existing record
+        cursor.execute(
+            "UPDATE public.assets SET name = %s, type = %s WHERE symbol = %s",
+            (name, asset_type, symbol[:9])
```

Comment:
symbol[:9]! cool! now only take 10 chars for the model, but 
the logic its repeat in:
+ line 29
+ line 35
DRY:)
its better put:
symbol = symbol[:9]
or not? and why?

Reply:
Although logic is present both in UPDATE and INSERT, I think it would crash if we left it in just INSERT, because there are tickers longer than the 10 char length